src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewType.java

```diff
@@ -7,16 +7,18 @@
     private String name;
     private String description;
     private CaseViewJurisdiction jurisdiction;
+    private String printableDocumentUrl;
```

Comment:
Could you please fix (or confirm the correctness of) the plurality of this attribute?

Reply:
Done.